src/translations/nbTranslations.json

```
@@ -463,6 +463,8 @@
         "channel_claim": "Kanaleierskap",
         "channel_owner": "Kanaleier",
         "claim_category_restriction": "Kategorier som begrensningen skal gjelde for",
+        "delete_channel_claim": "Fjern eierskap til kanal",
+        "delete_channel_claim_description": "<p>Er du sikker på at du vil fjerne eierskapet til <span>{{name}}</span>?</p",
```

Litt overrasket over at det faktisk kommer ut som riktig HTML, så antar de har noe logikk for å glatte over sånt 🤷
```suggestion
        "delete_channel_claim_description": "<p>Er du sikker på at du vil fjerne eierskapet til <span>{{name}}</span>?</p>",
```